src/components/sections/program/program-settings.js

```diff
@@ -138,36 +136,26 @@ class ProgramSettings extends React.Component {
     /**
      * handle onChange for global settings
      * */
-    handleChange = e => {
-        if (e.name === ENROLLMENT_DOWNLOAD) {
-            this.setState({
-                ...this.state,
-                disableSave: false,
-                [e.name]: e.value,
-            })
-        } else {
-            e.preventDefault()
+    handleChange = (e, key) => {
+        let name, value
 
-            if (e.target.name === SETTING_DOWNLOAD) {
-                if (
-                    e.target.value === GLOBAL ||
-                    e.target.value === PER_ORG_UNIT
-                ) {
-                    programData = GlobalProgramSpecial
-                } else {
-                    programData = GlobalProgram
-                }
-            }
+        typeof key === 'string'
+            ? ((name = key), (value = e.selected))
+            : ((name = e.name), (value = e.value))
 
-            this.setState({
-                ...this.state,
-                disableSave: false,
-                [e.target.name]: parseValueByType(
-                    e.target.name,
-                    e.target.value
-                ),
-            })
+        if (name === SETTING_DOWNLOAD) {
+            if (value === GLOBAL || value === PER_ORG_UNIT) {
+                programData = GlobalProgramSpecial
```

Comment:
I know this was here before, but it is **incorrect** - `programData` is defined as a singleton variable for this entire module, meaning there is only ever **one value for the entire app**.  Instead, this needs to be a state variable in the component itself (the state is set just below this).

Reply:
@Sharmyn28 let me know if there's a reason you made this a singleton